husky.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  hooks: {
+    'commit-msg': 'commitlint -E HUSKY_GIT_PARAMS',
+    'pre-commit': 'yarn lint',
+    'pre-push': 'yarn lint'
+  }
+}
```

Comment:
I prefer keep the husky config in the `package.json` because it's relative to the npm scripts.